sidestep_engine/data/preprocess.py

```
@@ -103,6 +102,103 @@ def _write_preprocess_meta(
+            data = torch.load(str(pt_file), weights_only=False, map_location="cpu")
+            latents = data.get("target_latents")
+            if latents is None:
+                continue
+            latents = latents.float()  # [T, C]
+            if running_sum is None:
+                running_sum = torch.zeros(latents.shape[-1], dtype=torch.float64)
+                running_sq_sum = torch.zeros(latents.shape[-1], dtype=torch.float64)
+            running_sum += latents.sum(dim=0).double()
+            running_sq_sum += (latents ** 2).sum(dim=0).double()
+            n_frames += latents.shape[0]
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)